README.md

```diff
@@ -129,6 +129,29 @@ inside of the image.
 podman run localhost/rag-content-openstack:latest ls /rag/vector_db/os_product_docs
 ```
 
+## Build with OKP content
+
+To include OKP content in the RAG, copy the non-paywalled OKP content into
```

Comment:
I think this is not necessary, you could copy everything, because your code will not add anything that's not part of `OKP_CONTENT_TYPES`, right?